Source/Krypton Components/Krypton.Ribbon/Palette/RibbonThemeManager.cs

```diff
@@ -17,38 +17,105 @@ namespace Krypton.Ribbon
     /// </summary>
     public class RibbonThemeManager
     {
-        #region Methods        
+        #region Theme Array        
         /// <summary>
-        /// Propagates the theme selector.
+        /// The supported themes
         /// </summary>
-        /// <param name="target">The target.</param>
-        public static void PropagateThemeSelector(KryptonRibbonGroupComboBox target)
-        {
-            foreach (var theme in ThemeManager.SupportedInternalThemeNames)
+        // TODO: This should use the list from Z:\GitHub\Krypton-Suite\Standard-Toolkit\Source\Krypton Components\Krypton.Toolkit\Converters\PaletteModeConverter.cs
+        private static readonly BiDictionary<string, PaletteMode> _supportedThemes = new(new Dictionary<string, PaletteMode>
             {
-                target.Items.Add(theme);
-            }
-        }
+                { @"Professional - System", PaletteMode.ProfessionalSystem },
```

Comment:
Can this be replaced by a single text entry somewhere, so that all the the themes names are only stored in a "Ssingle" file ?

Reply:
@Smurf-IV We can't use the `ThemeManager` in `Krypton.Toolkit`, as this would cause a circular dependency. However what we _could_ do is to create a 6th project, maybe called `Krypton.Shared` which would include single files/APIs which can be used in the other projects like `PI` etc?

Reply:
Or just a single file, which is then referenced across the projects that need it

Reply:
I'll turn it into a feature request, so it can be tracked